### DISCUSSION TITLE
Expose `AddInstances` for `InstancedStaticMeshComponent`

### DISCUSSION THREAD

**Discussion by realvictorprm** [nxrighthere/UnrealCLR#134] [Ideas]
Hi :)

I noticed that my performance is not that well if I add 10k transformations sequentially to my InstancedStaticMeshComponent. 
I then took a look whether the underlying UnrealEngine type really doesn't support adding multiple transforms with one call and stumpled apon the straightforward method `AddInstances` 😄 

Is there any obstacle for adding this method? If there isn't I would be happy to contribute the addition myself :)

For reference:
https://docs.unrealengine.com/en-US/API/Runtime/Engine/Components/UInstancedStaticMeshComponent/AddInstances/index.html

**Comment by nxrighthere** [comment]
Exposed in https://github.com/nxrighthere/UnrealCLR/commit/26cba91d348f530d4f92259748ee0bdf4bd6b54c.
